doc/source/check-licenses-action/index.rst

```diff
@@ -0,0 +1,44 @@
+Check licenses action
```

Comment:
Let us rename this to `Licenses actions` to be consistent with the rest of the headers. Rename the folder as well.